Queries.md

```diff
@@ -0,0 +1,419 @@
+	count(timestamp) filter (WHERE timestamp >= t.int AND timestamp < (t.int + interval '1 days'))
+FROM
+	t, (
+		SELECT *
+		FROM "event"
+		WHERE event = 'ChannelOpened' address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+		-- the ChannelOpened can be change for other events in order to obtain the other historgrams
+	) a
+GROUP BY
+  int
+order BY
+  int
+```
+
+
+## Daily network activity measured BY events in the network
+
+Network activity represents the number of events per day.
+
+```sql
+with t AS (
+	SELECT generate_series(mitstamp,matstamp,'1 days') AS int
+	FROM 
+	(
+		SELECT min(timestamp) mitstamp, now()::timestamp AS matstamp 
+	 	FROM "event" 
+		WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+	) a
+	order BY int asc
+)
+SELECT 
+	int AS timestampwindowstart,
+	count(timestamp) filter (WHERE timestamp >= t.int AND timestamp < (t.int + interval '1 days'))
+FROM
+	t, (
+		SELECT *
+		FROM "event"
+		WHERE address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474'
+	) a
+GROUP BY
+  int
+order BY
+  int
+```
+
+## Getting all participants addresses
+
+```sql
+with part AS (
+	SELECT  args->1#>'{hex}' AS participant1, args->2#>'{hex}' AS participant2
+	FROM "event"
+	WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+)
+SELECT participant1 AS participants
+FROM part
+union
+SELECT participant2 AS participants
+FROM part
+```
+
+
+## Counting the number of participants in the network
+
+```sql
+with part AS (
+	SELECT  args->1#>'{hex}' AS participant1, args->2#>'{hex}' AS participant2
+	FROM "event"
+	WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+)
+SELECT count(participants)
+FROM (
+	SELECT participant1 AS participants
+	FROM part
+	union
+	SELECT participant2 AS participants
+	FROM part
+) part
+```
+
+
+## AVG number of tokens used in a deposit
+
+```sql 
+Select AVG((args->2#>'{num}')::numeric)
+FROM "event"
+WHERE event = 'ChannelNewDeposit' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474';
+```
+
+
+## Histogram to show the deposited amount of tokens per day
+
+```sql
+with t AS (
+    SELECT generate_series(mitstamp,matstamp,'1 days') AS int
+    FROM 
+    (
+        SELECT min(timestamp) mitstamp, now()::timestamp AS matstamp 
+        FROM "event" 
+        WHERE event = 'ChannelOpened' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474' 
+    ) a
+    order BY int asc
+)
+SELECT timestampwindowstart, COALESCE(sum, 0) AS deposited
+FROM (
+	SELECT 
+		int AS timestampwindowstart,
+		SUM(deposited) filter (WHERE timestamp >= t.int AND timestamp < (t.int + interval '1 days'))
+	FROM
+		t, (
+			SELECT timestamp, (args->2#>'{num}')::numeric AS deposited 
+			FROM "event"
+			WHERE event = 'ChannelNewDeposit' AND address = '0xa5C9ECf54790334B73E5DfA1ff5668eB425dC474'
+		) a
+	GROUP BY
+	  timestampwindowstart
+	order BY
+	  timestampwindowstart
+) res
+```
+
+
+## Max AND Min deposit
```

Comment:
** Max and Min deposit** if this also made into a date histogram would be great.